generator/testdata/rust/gclient/golden/secretmanager/src/model.rs

```diff
@@ -864,6 +864,7 @@ pub struct SecretPayload {
     /// The CRC32C value is encoded as a Int64 for compatibility, and can be
     /// safely downconverted to uint32 in languages that support this type.
     /// https://cloud.google.com/apis/design/design_patterns#integer_types
+    #[serde(rename = "dataCrc32c")]
```

Comment:
This should be data_crc32c: https://github.com/googleapis/google-cloud-go/blob/47bb902e48695e5f0c46f7bee00d72ae5f794e1e/secretmanager/apiv1/secretmanagerpb/resources.pb.go#L973C97-L973C108